src/System.Runtime/tests/System/GC.cs

```diff
@@ -3,39 +3,46 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
-
 using Xunit;
 
-public class GCTests
+public static class GCTests
 {
+    private static bool s_is32Bits = IntPtrTests.TestSize == 4; // Skip IntPtr tests on 32-bit platforms
```

Comment:
Why isn't this just IntPtr.Size == 4?  Further, what you have here won't actually compile... TestSize is a method on the IntPtrTests class in this project.